manifests/init.pp

```diff
@@ -93,6 +93,7 @@
   }
 
   contain letsencrypt::renew
+  include letsencrypt::scripts
```

Comment:
If this is only used in `certonly`, should that have the include instead?

Reply:
done: 84f9150636e297c2447ed8c6f81d475c6917c2ae